go/vt/topo/etcd2topo/server.go

```diff
@@ -34,13 +34,24 @@ We follow these conventions within this package:
 package etcd2topo
 
 import (
+	"crypto/tls"
+	"flag"
+	"github.com/coreos/etcd/pkg/transport"
+	"io/ioutil"
 	"strings"
 	"time"
+	"vitess.io/vitess/go/vt/log"
 
 	"github.com/coreos/etcd/clientv3"
 	"vitess.io/vitess/go/vt/topo"
 )
 
+var (
+	certPath = flag.String("topo_etcd2_tls_cert", "", "the cert to use to connect to the etcd topo server, requires topo_etcd2_tls_key, enables TLS")
```

Comment:
please change the variable name to indicate this is a client cert ( same for the other two )

Reply:
Also change the doc string to say "client cert" and "client key".

Reply:
Done https://github.com/vitessio/vitess/pull/5496/commits/df22f4910d909a8b261f365de8224e1ee33621de.